### ISSUE TITLE
DataFrames dropped DataArrays for NullableArrays

### ISSUE THREAD

**Issue by phaverty** [armgong/rjulia#44]
DataFrames has switched from DataArrays and PooledDataArrays to NullableArrays and CategoricalArrays.  rjulia should switch too.  I think it would be very similar code.

**Comment by armgong** [comment]
yes, but the we need carefully read the dataframe docs ,and then change rjulia

**Comment by phaverty** [comment]
It looks like DataFrames will make the move in a few weeks.
https://discourse.julialang.org/t/announcement-dataframes-0-9-0-planned-for-february/266

**Comment by armgong** [comment]
I have new idea about rjulia ,  thinking using RCall.jl to write rjulia2, so we don't need so much c code to do conversion.

already write a prototype https://github.com/armgong/rjulia2   

I will try it during "Chinese new year" vacation ,  I will start this job from 2017-1-22 ,hope finish it in the vacation.

**Comment by phaverty** [comment]
Excellent, I think using RCall.jl as much as possible is the way to go.
The best thing about julia is not writing C anymore, I believe.  I'm
looking forward to experimenting with rjulia2.

Pete

____________________
Peter M. Haverty, Ph.D.
Genentech, Inc.
phaverty@gene.com
